requirements.txt

```diff
@@ -1,172 +1,191 @@
+    --hash=sha256:00f35fb0e7e2fc348cd091d25d36c6768b721463d12396d68a9491a8a5f91ecf
+newrelic==2.52.0.40 \
+    --hash=sha256:59b515bb7dbbcdd307f817ac38854e80839ebfa33c5db797f65450793c945c0d
+flake8==2.4.1 \
+    --hash=sha256:8fc4e865a29f3c6ecae401a7dce18247f8abd046d24e390e16af9976cc9c7aa8 \
+    --hash=sha256:2e7ebbe59d8c85e626e36d99f0db2f578394313d3f7ce9dc9f1da57ef6cd7537
+pep8==1.6.2 \
+    --hash=sha256:4aa129df8d9007b192bf82013f415533994652d7caa930d002687eb42a6c2a41 \
+    --hash=sha256:b8b7e35630b5539e26a197dfc6005be9e1e9a135496b377723a8ebc01b9bcbff
+pyflakes==0.9.2 \
+    --hash=sha256:05df584a29eeea9a2a2110dd362e53d04e0c4bb1754b4d71234f651917f3c2f0 \
+    --hash=sha256:02691c23ce699f252874b7c27f14cf26e3d4e82b58e5d584f000b7ab5be36a5f
+mccabe==0.3.1 \
+    --hash=sha256:bd6c080fb372aebcb0ce19e35ddac744f2abf5a7befa207db2d1097d48efe63a \
+    --hash=sha256:5f7ea6fb3aa9afe146d07fd6d5cedf788747d8b0c29e44732453c2b2db1e3d16
+ipython==3.2.0 \
+    --hash=sha256:5645b311f2ebcd2b54dcd993c0b6ad2eb5cad90f5f2fe3d45c184c843f44d407 \
+    --hash=sha256:e6790c69428fa6c3e0e4d633e26ad178970e468baffe03a98629054bdb84308e \
+    --hash=sha256:8e64b441e16298c08025b826126b2d7bc5c1776d2d2f071672166f615f327887 \
+    --hash=sha256:31f5d4e65ac0e72a7eafc31ecb95ab7c981bb49ed314b69311bca45c6ab8facb
+ipdb==0.8.1 \
+    --hash=sha256:1763d1564113f5eb89df77879a8d3213273c4d7ff93dcb37a3070cdf0c34fd7c
+docutils==0.12 \
+    --hash=sha256:dcebd4928112631626f4c4d0df59787c748404e66dda952110030ea883d3b8cd \
+    --hash=sha256:c7db717810ab6965f66c8cf0398a98c9d8df982da39b4cd7f162911eb89596fa
+Pygments==2.0.2 \
+    --hash=sha256:0a3a2265e1efb0defa722d1f429730dc9df975adc3c60ac7d2b432bdf03c041d \
+    --hash=sha256:5ded2fa9094fd7dfeb3da92636409fd702a0d07d606283504d7ee04401cee5cb \
+    --hash=sha256:7320919084e6dac8f4540638a46447a3bd730fca172afc17d2c03eed22cf4f51
+django-jinja==1.4.1 \
+    --hash=sha256:6f4c2e8c5ada17039ff1c1e38bc51bea9317af3676e28466148c8df5fc78f132
+defusedxml==0.4.1 \
+    --hash=sha256:cd551d5a518b745407635bb85116eb813818ecaf182e773c35b36239fc3f2478 \
+    --hash=sha256:1f7e2f6546caba6a52bcd6a6087df60c1cc63cdbc0d9ca3e5d517adcb5823d0c
+django-allauth==0.24.1 \
+    --hash=sha256:229a6a5a63ff55f9eb46d4dae3863ba775735bf60bfc16bf4e2042f214f18e1d
+oauthlib==1.0.3 \
+    --hash=sha256:ef4bfe4663ca3b97a995860c0173b967ebd98033d02f38c9e1b2cbb6c191d9ad
+python3-openid==3.0.6 \
+    --hash=sha256:833086e554515d58fe39f9326eae5c929ff448f0fce6350d45f622c9cc1591c6 \
+    --hash=sha256:8b82c6e8ac590b0433988138bf63d934dbf5cdcbad1643ec7e25cc0ce90ca50a
+requests==2.7.0 \
+    --hash=sha256:20f976cdce02a42b69ce80e9e03897a51814b36d448b37288546086ebc473146 \
+    --hash=sha256:398a3db6d61899d25fd4a06c6ca12051b0ce171d705decd7ed5511517b4bb93d
+requests_oauthlib==0.5.0 \
+    --hash=sha256:658d9aba85338be8c1d1532c9fb5807b381dc7166e469ff0f62fcaa4240d9eb8 \
+    --hash=sha256:c7d2ee6c2e8c58a0fd60c2d7d21ce93c8d2b22e7a9f232d4bf915326afb479af
+Pillow==3.2.0 \
```

Comment:
Wow, why so many hashes for Pillow? (also Pygments, psycopg2, pytz, etc) Wondering where you got them - guessing maybe they're for different platforms?

Tests pass, both on CircleCI and on my local machine. So, I may just merge this and go ahead to investigate for my own edification because I haven't caught up to how this stuff works with pip 8.1.2